Anotasi_Image/reviewer/templates/reviewer/isu_anotasi.html

```
@@ -0,0 +1,1113 @@
+        <svg xmlns="http://www.w3.org/2000/svg" width="22" height="22" viewBox="0 0 32 32">
+          <path fill="currentColor" d="M21.448 20A10.86 10.86 0 0 0 24 13a11 11 0 1 0-11 11a10.86 10.86 0 0 0 7-2.552L27.586 29L29 27.586ZM13 22a9 9 0 1 1 9-9a9.01 9.01 0 0 1-9 9"/><path fill="currentColor" d="M10 12H8v-2a2 2 0 0 1 2-2h2v2h-2zm8 0h-2v-2h-2V8h2a2 2 0 0 1 2 2zm-6 6h-2a2 2 0 0 1-2-2v-2h2v2h2zm4 0h-2v-2h2v-2h2v2a2 2 0 0 1-2 2"/>
+        </svg>
+      </span>
+    </div>
+  </div>
+  <div class="d-flex w-100 justify-content-center">
+    <div class="rounded" style="background-color:rgba(87,180,206,0.1);">
+      <div class="d-flex justify-content-senter align-items-center" style="font-size: 1.2rem;" >
+        <div class="btn" id="prevBtn">
+          <span style="color: white;">
+            <a href="{% if image_index > 1 %}{% url 'reviewer:isu_anotasi' image_index|add:'-2' %}{% else %}#{% endif %}" id="prevBtn" class="text-secondary text-decoration-none"> 
```

HTMLHint (1.5.0)

[error] The id value [ prevBtn ] must be unique.

(id-unique)